tools/src/test/scala/scala/scalanative/IncCompilationTest.scala

```diff
@@ -87,34 +141,10 @@ class IncCompilationTest extends codegen.CodeGenSpec with Matchers {
   )(implicit in: Scope): Config = {
     val classpath = makeClasspath(outDir)
     Config.empty
-      .withBasedir(outDir)
+      .withWorkdir(outDir)
       .withClassPath(classpath.toSeq)
       .withMainClass(entry)
       .withCompilerConfig(setupNativeConfig)
   }
 
-  private def nativeLink(
```

Comment:
This is more or less correct now. We only provide a `basedir` for the compiler to do everything and should not have a `workdir`. The `artifactPath` is now calculated by `NativeConfig` and not passed to `build` but returned by `build` or you can retrieve it from `NativeConfig` as well.